modules/getting-started/pages/glossary.adoc

```diff
@@ -0,0 +1,203 @@
+
+Mule® runtime engine (Mule® or Mule)::
+Java-based integration runtime engine for Anypoint Platform. uses a staged event-driven architecture (SEDA) to enqueue messages and process them inside of flows in separate stages. Mule is the engine for Anypoint Platform™ and the only runtime that combines data and application integration across legacy systems, SaaS applications, and APIs.
+
+= O
+
+organization::
+The container for everything in your Anypoint Platform account. 
+
+= P
+
+Pivotal Cloud Foundry::
+A cloud computing platform as a service (PaaS) provided by the company Pivotal. Anypoint Platform integrates with Pivotal Cloud Foundry, allowing you to deploy Mule applications to dynamically created virtual machines on your own private network. See deployment strategies. Previously referred to as PCF.
+
+policy::
+An object that controls authentication, access, consumption rates, and service level access (SLA) to API resources. Anypoint Platform provides both preconfigured and tools to create custom policies.
+
+= R
+
+RAML®::
+RESTful API Modeling Language (RAML) provides a specification language that you can use to define an API.
+
+Runtime Manager agent::
+The Runtime Manager agent mediates communication between the Anypoint Runtime Manager console and the Mule instances running on servers. Using the Mule agent, you can monitor and control Mule servers. Previously referred to as Runtime agent.
+
+runtime plane::
+The contract between the control plane and data plane for runtime control. In Anypoint Platform, the runtime plane consists of Mule runtime engine and runtime services managed with tools like Anypoint Runtime Fabric.
+
+= S
+
+secrets manager::
+Construct to store and control access to private keys, passwords, certificates, and other secrets.
+
+= T
+
+tokenization::
+Substituting a non-sensitive equivalent for a sensitive data element.
+
+transformer::
+A construct that transforms message payloads (data) to and from different types, such as the XSLT Transformer. Transformations can also be defined using DataWeave.
+
+transport::
+A construct that handles and carries messages on a specific messaging protocol, such as FTP. Several connectors incorporate transports.
+
+= U
+
+universal message object (UMO)::
+Obsolete name for service component. UMO is still visible in some MuleSoft APIs.
+
+= V
+
+Virtual Private Cloud::
+A virtual, private, and isolated network segment in the cloud that hosts CloudHub instances of Mule (CloudHub workers). Previously referred to as VPC.
```

Comment:
I think it's still referred to as VPC.

Reply:
Deleted. They want us to use Anypoint Virtual Private Cloud (VPC).